play-frontend-hmrc-play-30/src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/components/HmrcHeader.scala.html

```diff
@@ -31,57 +39,26 @@
-          </svg>
-        }
-        @productName match {
-          case Some(NonEmptyString(x)) => {
-            <span class="govuk-header__product-name">
-            @x
-            </span>
-          }
-          case _ => {}
-        }
+            @govukLogo(isRebrandEnabled, isTudorCrownEnabled, true, "govuk-header__logotype", Some("GOV.UK"))
+            @productName match {
+              case Some(NonEmptyString(x)) => {
+                <span class="govuk-header__product-name">@x</span>
```

Comment:
So, totally not a new thing, but the use of `x` as the decomposed param name here make me shudder...

Reply:
yeah, I believe previous person went with `x` because it was mapping on the `productName`, that that name couldn't be reused. Happy to change it to `name`

Reply:
Updated :)